test/unit/Signer.test.js

```diff
@@ -73,41 +73,43 @@ describe('Signer', () => {
         })
         it('should sign StreamMessageV30 correctly', async () => {
             const streamMessage = new StreamMessageV30(
-                [streamId, 0, timestamp, 0, null], [timestamp - 10, 0], StreamMessage.CONTENT_TYPES.JSON,
+                [streamId, 0, timestamp, 0, '', 'chain-id'], [timestamp - 10, 0], StreamMessage.CONTENT_TYPES.JSON,
```

Comment:
If we test with `publisherId` being the empty string, the test wouldn't catch a case where the `publisherId` is accidentally ignored when concatenating the payload to be signed!

Reply:
Here `publisherId` can be anything because it is going to be set by the `Signer` to be `signer.address` (which is the `publisherId`). This is why on the next line the expected `payload` is using `signer.address` as `publisherId`.